NSURLSessionCancellation/ViewController.swift

```diff
@@ -27,9 +27,9 @@ class ViewController: UIViewController {
 
         // Download data
 
-        session.dataTask(with: url, completionHandler: { (data, response, error) in
+        session.dataTask(with: url) { (data, response, error) in
```

Comment:
I do remove the parentheses. In addition to being the convention in the RW style guide (though as you said, not an explicit rule), it also appears to be convention on Apple's GitHub repos as well, from what I've seen. Unless parentheses are explicitly required, the convention appears to be not to add them.